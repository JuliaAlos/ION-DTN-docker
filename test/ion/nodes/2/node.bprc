1
a scheme ipn 'ipnfw' 'ipnadminep'
a endpoint ipn:2.0 x
a endpoint ipn:2.1 q
a endpoint ipn:2.2 x
a endpoint ipn:2.64 x
a endpoint ipn:2.65 x
a protocol udp 1400 100
a induct udp 192.168.0.102:2111 udpcli
a induct udp 192.168.0.102:2114 udpcli
a outduct udp 192.168.0.101:1112 udpclo
a outduct udp 192.168.0.104:4112 udpclo
a plan ipn:1.0
a planduct ipn:1.0 udp 192.168.0.101:1112
a plan ipn:3.0
a planduct ipn:1.0 udp 192.168.0.101:1112
a planduct ipn:4.0 udp 192.168.0.104:4112
a plan ipn:4.0
a planduct ipn:4.0 udp 192.168.0.104:4112
a plan ipn:5.0
a planduct ipn:4.0 udp 192.168.0.104:4112
s
