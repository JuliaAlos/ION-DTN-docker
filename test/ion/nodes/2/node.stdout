[i] admin pgm using SDR parm overrides from bench.ionconfig.
wmKey:           0
wmSize:          40000000
wmAddress:       0
sdrName:        ''
sdrWmSize:       0
configFlags:     1
heapWords:       20000000
heapKey:         -1
logSize:         0
logKey:          -1
pathName:       '/tmp'
Initializing semaphores to use: Posix Named Semaphores - max : 2048
Stopping ionadmin.
Stopping ionsecadmin.
Stopping ionadmin.
Stopping bpadmin.
at line 600 of ici/library/ion.c, No node has been initialized in this directory. (/ION-DTN/ion-interface/test/ion/nodes/2)
at line 986 of ici/library/ion.c, Failed checking node list parms.
at line 1865 of bpv7/library/libbpP.c, BP can't attach to ION.
at line 1240 of bpv7/ipn/ipnfw.c, ipnfw can't attach to BP.
at line 600 of ici/library/ion.c, No node has been initialized in this directory. (/ION-DTN/ion-interface/test/ion/nodes/2)
at line 986 of ici/library/ion.c, Failed checking node list parms.
at line 653 of ici/utils/ionwarn.c, ionwarn can't attach to ION.
at line 600 of ici/library/ion.c, No node has been initialized in this directory. (/ION-DTN/ion-interface/test/ion/nodes/2)
at line 986 of ici/library/ion.c, Failed checking node list parms.
at line 1865 of bpv7/library/libbpP.c, BP can't attach to ION.
at line 534 of bpv7/daemon/bpclock.c, bpclock can't attach to BP.
at line 600 of ici/library/ion.c, No node has been initialized in this directory. (/ION-DTN/ion-interface/test/ion/nodes/2)
at line 986 of ici/library/ion.c, Failed checking node list parms.
at line 455 of ici/daemon/rfxclock.c, rfxclock can't attach to ION.
at line 600 of ici/library/ion.c, No node has been initialized in this directory. (/ION-DTN/ion-interface/test/ion/nodes/2)
at line 986 of ici/library/ion.c, Failed checking node list parms.
at line 1865 of bpv7/library/libbpP.c, BP can't attach to ION.
at line 403 of bpv7/daemon/cpsd.c, cpsd can't attach to BP.
at line 600 of ici/library/ion.c, No node has been initialized in this directory. (/ION-DTN/ion-interface/test/ion/nodes/2)
at line 986 of ici/library/ion.c, Failed checking node list parms.
at line 1865 of bpv7/library/libbpP.c, BP can't attach to ION.
at line 152 of bpv7/udp/udpcli.c, udpcli can't attach to BP.
at line 600 of ici/library/ion.c, No node has been initialized in this directory. (/ION-DTN/ion-interface/test/ion/nodes/2)
at line 986 of ici/library/ion.c, Failed checking node list parms.
at line 1865 of bpv7/library/libbpP.c, BP can't attach to ION.
at line 446 of bpv7/daemon/bpclm.c, bpclm can't attach to BP.
at line 600 of ici/library/ion.c, No node has been initialized in this directory. (/ION-DTN/ion-interface/test/ion/nodes/2)
at line 986 of ici/library/ion.c, Failed checking node list parms.
at line 653 of ici/utils/ionwarn.c, ionwarn can't attach to ION.
at line 600 of ici/library/ion.c, No node has been initialized in this directory. (/ION-DTN/ion-interface/test/ion/nodes/2)
at line 986 of ici/library/ion.c, Failed checking node list parms.
at line 1865 of bpv7/library/libbpP.c, BP can't attach to ION.
at line 446 of bpv7/daemon/bpclm.c, bpclm can't attach to BP.
at line 600 of ici/library/ion.c, No node has been initialized in this directory. (/ION-DTN/ion-interface/test/ion/nodes/2)
at line 600 of ici/library/ion.c, No node has been initialized in this directory. (/ION-DTN/ion-interface/test/ion/nodes/2)
at line 986 of ici/library/ion.c, Failed checking node list parms.
at line 1865 of bpv7/library/libbpP.c, BP can't attach to ION.
at line 123 of bpv7/udp/udpclo.c, udpclo can't attach to BP.
at line 986 of ici/library/ion.c, Failed checking node list parms.
at line 1865 of bpv7/library/libbpP.c, BP can't attach to ION.
at line 446 of bpv7/daemon/bpclm.c, bpclm can't attach to BP.
at line 600 of ici/library/ion.c, No node has been initialized in this directory. (/ION-DTN/ion-interface/test/ion/nodes/2)
at line 986 of ici/library/ion.c, Failed checking node list parms.
at line 1865 of bpv7/library/libbpP.c, BP can't attach to ION.
at line 123 of bpv7/udp/udpclo.c, udpclo can't attach to BP.
at line 600 of ici/library/ion.c, No node has been initialized in this directory. (/ION-DTN/ion-interface/test/ion/nodes/2)
at line 986 of ici/library/ion.c, Failed checking node list parms.
at line 1865 of bpv7/library/libbpP.c, BP can't attach to ION.
at line 121 of bpv7/daemon/bptransit.c, bptransit can't attach to BP.
at line 600 of ici/library/ion.c, No node has been initialized in this directory. (/ION-DTN/ion-interface/test/ion/nodes/2)
at line 600 of ici/library/ion.c, No node has been initialized in this directory. (/ION-DTN/ion-interface/test/ion/nodes/2)
at line 986 of ici/library/ion.c, Failed checking node list parms.
at line 1865 of bpv7/library/libbpP.c, BP can't attach to ION.
at line 446 of bpv7/daemon/bpclm.c, bpclm can't attach to BP.
at line 986 of ici/library/ion.c, Failed checking node list parms.
at line 1865 of bpv7/library/libbpP.c, BP can't attach to ION.
at line 150 of bpv7/ipn/ipnadminep.c, ipnadminep can't attach to BP.
at line 600 of ici/library/ion.c, No node has been initialized in this directory. (/ION-DTN/ion-interface/test/ion/nodes/2)
at line 986 of ici/library/ion.c, Failed checking node list parms.
at line 1865 of bpv7/library/libbpP.c, BP can't attach to ION.
at line 152 of bpv7/udp/udpcli.c, udpcli can't attach to BP.
